src/main/java/org/pethotel/HeavenForPets/controllers/HotelController.java

```diff
@@ -46,17 +48,21 @@ public void updateDiscountAtOwner(@PathVariable String lastName, @PathVariable i
         ownerService.updateDiscountAtOwner(lastName, newDiscount);
     }
 
-
     @GetMapping("/getAllRoomsByType/{petType}")
     public List<Room> getAllRoomsByType(@PathVariable String petType){
         return roomService.getAllRoomsByType(petType);
     }
 
-    @GetMapping("/get")
-    public List<Pet> getPets(){
+    @GetMapping("/getAnimals")
```

Comment:
Maybe, if the name of URL was changed, you can consider rename the methot to getAminals